src/events.ts

```diff
@@ -56,15 +56,15 @@ function unbindable(el: EventTarget, eventName: string, callback: MouchBind, opt
 
 function startDragOrDraw(s: State): MouchBind {
   return e => {
-    if (isRightButton(e) && s.draggable.current) drag.cancel(s);
-    else if ((e.shiftKey || isRightButton(e)) && s.drawable.enabled) draw.start(s, e);
+    if (isRightClick(e) && s.draggable.current) drag.cancel(s);
```

Comment:
Fixed, but now I'm concerned that people will hit `ctrl` **before** beginning to drag the pawn and will lose valuable time accidentally drawing. I like being able to right-click using control in the board editor, but this PR might be best to decline now that I know about this feature.